src/class_loader_core.cpp

```diff
@@ -30,12 +30,15 @@
 #include "class_loader/class_loader_core.hpp"
 #include "class_loader/class_loader.hpp"
 
+#include <boost/filesystem.hpp>
+
 #include <Poco/SharedLibrary.h>
 
 #include <cassert>
 #include <cstddef>
 #include <string>
 #include <vector>
+#include <iostream>
```

Comment:
What is this include for?

Reply:
Leftover from debugging, I've removed it and ensured the DCO check passes.